spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/mapping/MappingRuleInterpreterBroadcast.scala

```diff
@@ -47,7 +47,7 @@ case class MappingRuleInterpreterBroadcast(rule: MappingConformanceRule, conform
     val broadcastedMt = spark.sparkContext.broadcast(mt)
     val errorUDF = BroadcastUtils.getErrorUdf(broadcastedMt, rule.allOutputColumns().keys.toSeq, mappings)
 
-    if (rule.additionalColumns.isEmpty) {
+    if (rule.additionalColumns.isEmpty || rule.additionalColumns.contains(Map())) {
```

Comment:
```suggestion
    if (additionalColumns.getOrElse(Map()).isEmpty) {
```

Btw. additionalColumns emptyness is also check at `MappingRuleInterpreterGroupExplode:54` and at `DynamicInterpreter:239` only using `.isEmpty`, so I am wondering if this is enough or perhaps it should also be checked for `Some(Map())`, too.

Reply:
And the condition at `DynamicInterpreter:239` is non-issue? If so, why btw?

Reply:
I forgot about it, I will push that too